### DISCUSSION TITLE
Having a dedicate relation suggestor trigger of each hierarchy

### DISCUSSION THREAD

**Discussion by ooker777** [SkepticMystic/breadcrumbs#355] [Ideas]
For example, if I press `cc`, then I will have suggestions for hierarchy `up`, `down`, `next`. If I press `dd`, then I will have suggestions for hierarchy `parent`, `children`, `sibling`. In the current version all fields are suggested in one place, making it cluttering

**Comment by ooker777** [comment]
I use Various Complement and it works like a charm. Here is my dictionary:
```
up:: [[|]]		ccu
down:: [[|]]		ccd
next:: [[|]]		ccn

parent:: [[|]]		ddp
children:: [[|]]		ddc
sibling:: [[|]]		dds
```

**Comment by michaelpporter** [comment]
This is really an editor-autocomplete concern rather than something Breadcrumbs itself does — BC reads your fields from frontmatter/properties, but it doesn't provide a typing-suggestion engine.

As @ooker777 notes above, a dedicated completion plugin handles this well. **Various Complement** (or Templater snippets) lets you bind triggers like `ccu` → `up:: [[|]]`, `ddp` → `parent:: [[|]]`, giving you exactly the per-group triggers described here. Obsidian's native property autocomplete also scopes by property name.

Since it's out of scope for Breadcrumbs and well covered by existing tools, closing this.
